pkg/controller/loaders/fileLoader.go

```diff
@@ -46,7 +47,7 @@ func (loader *fileLoader) SetUUID(devUUID uuid.UUID) {
 }
 
 //ProcessExisting for observe existing files
```

Comment:
Also it will be good to be more verbose. If I understand correctly - this is to process logfiles that already exist locally on Eden.

Reply:
This function is for processing existing elements (not to follow new ones) in different sources. It is similar to `cat` function against `tail -f`